### DISCUSSION TITLE
node_modules\react-native-reanimated\android\.cxx\Debug\n3v6x2g3\arm64-v8a\.cmake\api\v1\query\client-agp\codemodel-v2 (Access is denied)

### DISCUSSION THREAD

**Discussion by tuanntvcu** [software-mansion/react-native-reanimated#5580] [Q&A]
Please help me!
![image](https://github.com/software-mansion/react-native-reanimated/assets/44677061/d6b686b4-982a-445c-972f-cefeff309632)

**Comment by webdevineer** [comment]
> Please help me! ![image](https://private-user-images.githubusercontent.com/44677061/296528931-d6b686b4-982a-445c-972f-cefeff309632.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0MDE4NDUsIm5iZiI6MTcxMTQwMTU0NSwicGF0aCI6Ii80NDY3NzA2MS8yOTY1Mjg5MzEtZDZiNjg2YjQtOTgyYS00NDVjLTk3MmYtY2VmZWZmMzA5NjMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDIxMTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzMTZlZDdkOTE1ZTZjMjVjMGU2ZGY1ZTJlMGUwMmEyYTQ1NTY1YWZhY2Q1Yzg4ZjRkMDY4ZmYxOTRmZTZiYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jScTTCweFDUcG1LHYthiN2Sw0QE0VM5Cq5c0kGBNG6w)

Where you able to solve this issue; how? Have been stuck on this for over a 2 weeks now. No suggessions seem to work for me.

**Comment by MatiPl01** [comment]
Hey!
It looks like there is a problem with your Windows user permissions. Does it happen only when you try to use `react-native-reanimated` or you had similar problems with other libraries as well?

**Reply by alham0046** [reply]
same issue here. and yes it happens only with reanimated

**Comment by giscarcor** [comment]
could you solve it @tuanntvcu @webdevineer ? I have the same issue
